wgpu-hal/src/gles/egl.rs

```diff
@@ -1015,6 +1049,7 @@ impl crate::Surface<super::Api> for Surface {
                 attributes.push(egl::ATTRIB_NONE as i32);
 
                 // Careful, we can still be in 1.4 version even if `upcast` succeeds
+                #[cfg(not(feature = "emscripten"))]
                 let raw_result = match self.egl.instance.upcast::<egl::EGL1_5>() {
```

Comment:
this is done because `egl::EGL1_5` isn't on emscripten? any idea why this is the case?
I see "1_5" feature enabled by default - https://github.com/timothee-haudebourg/khronos-egl/blob/76d7d917f1c5b40061596653a16f0340683b5e0b/Cargo.toml#L14

Reply:
Not 100% sure how features works, but we use "static" feature that not implies 1_5 or any.

Reply:
Ok, here is what I see. It's not that EGL1_5 isn't available, but rather that a static instance can't be `upcast` at all...

Reply:
How I can use the same default branch (_ =>) and not failing on unresolved EGL1_5?

Reply:
Ok, I have an idea